drivers/gles3/shader_gles3.cpp

```diff
@@ -585,6 +585,19 @@ bool ShaderGLES3::_load_from_cache(Version *p_version) {
 			Version::Specialization specialization;
 
 			specialization.id = glCreateProgram();
+			if (feedback_count) {
+				Vector<const char *> feedback;
+				for (int i = 0; i < feedback_count; i++) {
```

Comment:
The variable `i` is already used in the outer loop, so this is raising a shadowing warning. You should use another letter (e.g. `k` since `j` is also already used).